webapp-django/crashstats/settings/local.py-dist

```diff
@@ -168,3 +168,19 @@ SECRET_KEY = 'you must change this'
 #           'component=General&groups=client-services-security',
 #    'label': 'File a bug in bugzilla'
 #}
+#
+# We're adding configman to django, and so we need to specify a path 
+# to the .ini files needed. The setting below is appropriate for local
+# development where you invoke manage.py in the webapp-django/ directory
+# this configuration is for our travis environment
+import os
+build_path = os.environ.get(
+    'TRAVIS_BUILD_DIR',
+    os.environ.get(
+        'WORKSPACE',  # Jenkins
+        '..'  # development where webapp is in a sub-directory of socorro
+    )
+)
+
+DATASERVICE_INI = build_path + '/config/dataservice.ini'
+assert os.path.isfile(DATASERVICE_INI), DATASERVICE_INI
```

Comment:
Basically, `settings/local.py` is just for exceptional overrides and things, like passwords, that can never be in `settings/base.py`.